examples/example-rtm-client-datastore.js

```diff
@@ -7,8 +7,8 @@ var MemoryDataStore = require('@slack/client').MemoryDataStore;
 var token = process.env.SLACK_API_TOKEN || '';
 
 var rtm = new RtmClient(token, {
-  logLevel: 'info', // check this out for more on logger: https://github.com/winstonjs/winston
-  dataStore: new MemoryDataStore({}) // pass a new MemoryDataStore instance to cache information
```

Comment:
Why these changes? This seems outside the scope of your PR's discussion (removing invalid options from documentation)